### ISSUE TITLE
changing emoji set

### ISSUE THREAD

**Issue by dragonwocky** [dragonwocky/notion-enhancer#8]
Replacing the Twitter emoji set used with e.g. Apple or Microsoft's emojis. Theoretical implementation: replace the CSS background set to Notion's `<image class="notion-emoji">` elements with the location of the emoji (found in the `alt` attr) looked up in a spritesheet.

Would also have a switch for "use system default" - code for this already written/tested/working.

**Comment by dragonwocky** [comment]
```js
const observer = new MutationObserver((list, observer) => {
  document.querySelectorAll('.notion-record-icon .notion-emoji').forEach(el => {
    el.outerHTML = <span style="font-size:0.8em; position: relative; bottom: 0.2em; right: 0.05em">${el.getAttribute('alt')}</span>
  });
  document.querySelectorAll('.notion-emoji').forEach(el => {
    el.outerHTML = <span>${el.getAttribute('alt')}</span>
  });
});
observer.observe(document, { childList: true, subtree: true })
```

snippet for potentially system default implementation

**Comment by dragonwocky** [comment]
![image](https://user-images.githubusercontent.com/16874139/90256477-28865700-de89-11ea-877c-67d377c2b84b.png)

**Comment by dragonwocky** [comment]
released in v0.8.0
